src/Lightweight/DataBinder/UnicodeConverter.cpp

```diff
@@ -112,11 +173,7 @@ std::u16string ToUtf16(std::string const& localeInputString)
     return { reinterpret_cast<char16_t const*>(wideString.data()),
              reinterpret_cast<char16_t const*>(wideString.data() + wideString.size()) };
 #else
-    std::locale sys_locale("");
-    std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> utf16conv;
-    auto const result = utf16conv.from_bytes(localeInputString);
-    return { reinterpret_cast<char16_t const*>(result.data()),
-             reinterpret_cast<char16_t const*>(result.data() + result.size()) };
+    return Utf8ToUtf16(localeInputString);
```

Comment:
```suggestion
    return ToUtf16(std::u8string_view(localeINputString));
```
Then this line would be something like this.

Reply:
compiler does not allow to construct `std::u8string_view(localeINputString)`